pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -188,8 +192,8 @@ var _ = Describe("Pod Network", func() {
 		mockNetwork.EXPECT().AddrAdd(bridgeTest, masqueradeGwAddr).Return(nil)
 		mockNetwork.EXPECT().AddrAdd(bridgeTest, masqueradeIpv6GwAddr).Return(nil)
 		mockNetwork.EXPECT().StartDHCP(masqueradeTestNic, masqueradeGwAddr, api.DefaultBridgeName, nil)
-		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(api.DefaultVMCIDR).Return("10.0.2.1/30", "10.0.2.2/30", nil)
-		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(api.DefaultVMIpv6CIDR).Return("fd10:0:2::1/120", "fd10:0:2::2/120", nil)
+		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(ipv4CidrStr).Return(masqueradeGwStr, masqueradeVmStr, nil)
+		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(ipv6CidrStr).Return(masqueradeIpv6GwStr, masqueradeIpv6VmStr, nil)
```

Comment:
same

Reply:
Please close this conversation in favor of the one opened on line 111.